pytorch/inference/docker/1.7.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,130 @@
+FROM ubuntu:16.04
```

Comment:
Could you verify if this should remain ubuntu 16.04?

Reply:
Yes we can. Just need to first have the PR green for PT1.7 official release first.

Reply:
Thanks.